pkg/sqlutil/pg/pg.go

```diff
@@ -0,0 +1,35 @@
+package pg
+
+import (
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/scylladb/go-reflectx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func NewSqlxDB(t testing.TB, dbURL string) *sqlx.DB {
```

Comment:
Should the name say test? Maybe instead of sqlx?
```suggestion
func NewTestDB(t testing.TB, dbURL string) *sqlx.DB {
```

Reply:
We can change it to whatever we want here... but I'd prefer not to change the name of the wrapper function in the core repo that calls this. Since that's called in unit tests all throughout the codebase, the CODEOWNERS requirement for the search-and-replace would be a nightmare